packages/vue/src/components/confirm-sign-up.vue

```diff
@@ -117,9 +118,11 @@ export default defineComponent({
     const {
       value: { context },
     } = state;
+    const actorState = computed(() => getActorState(state.value));
+    const actorContext = actorState?.value.context;
 
     let [primaryAlias] = useAliases(context?.config?.login_mechanisms);
-    if (!context?.formValues?.confirm_password) {
+    if (!actorContext.formValues?.confirm_password) {
       primaryAlias = 'username';
     }
```

Comment:
I don't believe this needs to exist anymore since we don't need to look up the primaryAlias any longer since the value is always being stored in `authAttributes.username` in the signUp Actor

Reply:
Sorry, this is a typo in my end. SignUp still stores usernames as email/phone_number depending on `primaryAlias`. I'll update the code to reflect this.

Reply:
In future PRs, we should consider storing that value as `username` on signUp, then we can disambiguate it during submit.

Reply:
Wait nvm!! Scratch all I said. You're right, the value is always passed from `authAttributes.username`. Thanks @ErikCH 🎉